### ISSUE TITLE
An error occurred while sending transation: -26: 18: bad-txns-inputs-spent

### ISSUE THREAD

**Issue by goodc0re** [yura-pakhuchiy/znode-tool#3]
When trying to spend XZC the following error occurs after signing the transaction with Trezor and trying to send it: 

An error occurred while sending transation: -26: 18: bad-txns-inputs-spent

**Comment by yura-pakhuchiy** [comment]
Does inputs have 100 confirmations each (I assume you are trying to transfer znode earnings)? Also try pressing "Reload" button in "Transfer funds dialog" button (maybe you are trying to spend already spent output). If nothing from above helps - post signed raw transaction here or send it to me privately pakhuchiy@gmail.com

**Comment by goodc0re** [comment]
The unspendable inputs have some things in common:

They all have 104213 confirmations. 
They all have an incorrect date: 31/10/14 18:24

**Comment by hakonamatata** [comment]
I have the same issue

It shows 15 zcoin with 110826 confirmations dated 31-oct-14

and I get the error mentioned above

**Comment by hakonamatata** [comment]
> 
> 
> @hakonamatata This information is not enough to reproduce. Could you send me your znode address and hashes of transactions (which you are trying to spend according to the tool and the real one if it differs). [pakhuchiy@gmail.com](mailto:pakhuchiy@gmail.com)

Sent you an email now. Let me know if you need any more information

**Comment by yura-pakhuchiy** [comment]
@hakonamatata Thank you for the provided data. I've tried to reproduce locally, but the tool displays correct transactions for your address on my system. The output from the tool on my system matches https://chainz.cryptoid.info/xzc/ and Electrum-XZC outputs. Insight provides different output, but I guess it is a bug in insight.

The tool uses `getaddressutxos` command in order to receive list of unspent UTXOs. @hakonamatata and @goodc0re could you try this command directly on your RPC node and check whether output is correct. If output is incorrect, then this is `zcoind` bug and should be reported to https://github.com/zcoinofficial/zcoin. Here is sample command (replace address with your Znode address):
```
zcoin-cli getaddressutxos '{"addresses": ["a1HwTdCmQV3NspP2QqCGpehoFpi8NY4Zg3"]}'
```
NOTE: It is important to issue this command on your RPC node for the tool, not on any other wallet.

**Comment by yura-pakhuchiy** [comment]
I received confirmation from @hakonamatata that `getaddressutxos` returns invalid data. This is bug in zcoind and should be reported to https://github.com/zcoinofficial/zcoin.

Following workarounds can be tried until this bug is fixed in zcoind:
1. start zcoind with `-reindex` option once (and let it finish reindex).
2. Upgrade/downgrade zcoind and sync blockchain from scratch.

**Comment by yura-pakhuchiy** [comment]
Upstream issue: https://github.com/zcoinofficial/zcoin/issues/311

**Comment by yura-pakhuchiy** [comment]
@goodc0re @hakonamatata Fix is released in Zcoin Core 0.13.7.4
